infoblox_client/objects.py

```diff
@@ -742,13 +742,21 @@ def ip(self, ip):
 
 class FixedAddressV4(FixedAddress):
     _infoblox_type = 'fixedaddress'
-    _fields = ['ipv4addr', 'mac', 'network_view', 'extattrs', 'network']
+    _fields = ['ipv4addr', 'mac', 'network_view', 'extattrs', 'network','options']
```

Comment:
nit: need a space after comma to make pep8 checks happy, also this line is longer than 79 characters, so need to move 'options' to next line.

> infoblox_client/objects.py:745:72: E231 missing whitespace after ','
> infoblox_client/objects.py:745:80: E501 line too long (82 > 79 characters)

Just run 'make lint' to make sure your changes passes pep8/flake8 rules.

P.S. shame on me that I still have not included flake checks into Travis CI (created #79 for that).